project/Build.scala

```diff
@@ -257,7 +258,9 @@ object ScaldingBuild extends Build {
       "org.scalacheck" %% "scalacheck" % scalaCheckVersion % "test"
     ),
     testFrameworks += new TestFramework("org.scalameter.ScalaMeterFramework"),
-    parallelExecution in Test := false
+    parallelExecution in Test := false,
+    skip in Test := true,
```

Comment:
I thought this is how you skip tests in sbt but `./sbt clean test` under root still ran the tests for this module. What am i missing?